udata/core/spatial/models.py

```diff
@@ -334,3 +334,11 @@ def top_label(self):
     def handled_zones(self):
         """Return only zones with a dedicated page."""
         return [zone for zone in self.zones if zone.handled_level]
+
+    def clean(self):
```

Comment:
This ended up not being the cause of the error but seems important in a logical way. Do we keep it?